test/runtime/monitor.go

```diff
@@ -123,6 +123,29 @@ var _ = Describe("RuntimeValidatedMonitorTest", func() {
 			Expect(res.Output().String()).Should(ContainSubstring(v))
 			vm.SampleContainersActions(helpers.Delete, helpers.CiliumDockerNetwork)
 		}
+
+		By(fmt.Sprintf("all types together"))
+		command := "cilium monitor -v"
+		for k := range eventTypes {
+			command = command + " --type " + k
+		}
+
+		ctx, cancel := context.WithCancel(context.Background())
+		res = vm.ExecContext(ctx, command)
+		vm.SampleContainersActions(helpers.Create, helpers.CiliumDockerNetwork)
+		areEndpointsReady := vm.WaitEndpointsReady()
+		Expect(areEndpointsReady).Should(BeTrue())
+
+		vm.ContainerExec(helpers.App1, helpers.Ping(helpers.Httpd1))
+		helpers.Sleep(10)
+		cancel()
+
+		Expect(res.CountLines()).Should(BeNumerically(">", 3))
+		output := res.Output().String()
+		for _, v := range eventTypes {
+			Expect(output).Should(ContainSubstring(v))
+		}
+		vm.SampleContainersActions(helpers.Delete, helpers.CiliumDockerNetwork)
```

Comment:
You don't need to do this here. In fact is a bad practice, all the cleanups should happen after `vm.ReportFailed`